app/uk/gov/hmrc/helptosavefrontend/controllers/email/AccountHolderUpdateEmailAddressController.scala

```diff
@@ -140,15 +141,15 @@ class AccountHolderUpdateEmailAddressController @Inject() (val helpToSaveService
         (enrolmentStatus, maybeEmail) match {
           case (EnrolmentStatus.NotEnrolled, _) ⇒
             // user is not enrolled in this case
-            auditor.sendEvent(SuspiciousActivity(nino, "missing_enrolment"), nino)
+            auditor.sendEvent(SuspiciousActivity(Some(nino), "missing_enrolment"), nino)
```

Comment:
Pls change logger.warn string to "SuspiciousActivity: missing enrolment for user's nino"